### PR TITLE
dumpling-overview: Suggest 256 MB or less as file size

### PR DESCRIPTION
**Pull request by Joyinqin**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
This is because lightning is suggesting this for non-CSV files.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/5328
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dumpling-overview.md [hunk 1]
```diff
@@ -248,7 +248,8 @@ Dumpling 也可以通过 `-B` 或 `-T` 选项导出特定的数据库/数据表
 
 默认情况下，导出的文件会存储到 `./export-<current local time>` 目录下。常用选项如下：
 
-- `-t` 用于指定导出的线程数。增加线程数会增加 Dumpling 并发度，但也会加大数据库内存消耗，因此不宜设置过大。
+- `-o` 用于选择存放导出文件的目录。
```
Comment (TomShawn) [inline dumpling-overview.md:251]: Please align the latest version of this doc: https://github.com/pingcap/docs/blob/master/dumpling-overview.md#improve-export-efficiency-through-concurrency

Comment (Joyinqin) [reply dumpling-overview.md]: Updated~

File: dumpling-overview.md [hunk 2]
```diff
@@ -249,6 +249,7 @@ Dumpling 也可以通过 `-B` 或 `-T` 选项导出特定的数据库/数据表
 默认情况下，导出的文件会存储到 `./export-<current local time>` 目录下。常用选项如下：
 
 - `-t` 用于指定导出的线程数。增加线程数会增加 Dumpling 并发度，但也会加大数据库内存消耗，因此不宜设置过大。
+- `-F` 选项用于指定单个文件的最大大小（此处单位是 `MiB`；可以接受类似 `5GiB` 或 `8KB` 的输入）。如果你打算使用 TiDB Lightning 将该文件加载到 TiDB 实例中，建议将其值保持在 256 MiB 或更小。
```
Comment (TomShawn) [inline dumpling-overview.md:252]:
```suggestion
- `-F` 选项用于指定单个文件的最大大小（单位为 `MiB`，可接受类似 `5GiB` 或 `8KB` 的输入）。如果你想使用 TiDB Lightning 将该文件加载到 TiDB 实例中，建议将 `-F` 选项的值保持在 256 MiB 或以下。
```

**Comment by lichunzhu** [comment]
To me `-F` isn't a `concurrency` option. Should we still put this line here or somewhere else?

**Comment by TomShawn** [comment]
> To me `-F` isn't a `concurrency` option. Should we still put this line here or somewhere else?

This PR is translated from https://github.com/pingcap/docs/pull/5328. If this is not the perfect place for `-F`, do you have any good idea where to put it?

**Comment by lichunzhu** [comment]
> > To me `-F` isn't a `concurrency` option. Should we still put this line here or somewhere else?
> 
> This PR is translated from [pingcap/docs#5328](https://github.com/pingcap/docs/pull/5328). If this is not the perfect place for `-F`, do you have any good idea where to put it?

Put to [here](https://docs.pingcap.com/tidb/stable/dumpling-overview#export-to-sql-files)? Not so sure.

**Comment by TomShawn** [comment]
> > > To me `-F` isn't a `concurrency` option. Should we still put this line here or somewhere else?
> > 
> > 
> > This PR is translated from [pingcap/docs#5328](https://github.com/pingcap/docs/pull/5328). If this is not the perfect place for `-F`, do you have any good idea where to put it?
> 
> Put to [here](https://docs.pingcap.com/tidb/stable/dumpling-overview#export-to-sql-files)? Not so sure.

No problem.

File: dumpling-overview.md [hunk 3]
```diff
@@ -249,6 +249,7 @@ Dumpling 也可以通过 `-B` 或 `-T` 选项导出特定的数据库/数据表
 默认情况下，导出的文件会存储到 `./export-<current local time>` 目录下。常用选项如下：
 
 - `-t` 用于指定导出的线程数。增加线程数会增加 Dumpling 并发度，但也会加大数据库内存消耗，因此不宜设置过大。
+- `-F` 选项用于指定单个文件的最大大小（单位为 `MiB`，可接受类似 `5GiB` 或 `8KB` 的输入）。如果你想使用 TiDB Lightning 将该文件加载到 TiDB 实例中，建议将 `-F` 选项的值保持在 256 MiB 或以下。
```
Comment (TomShawn) [inline dumpling-overview.md:252]: @Joyinqin Please move this line to line 72 and also update the English version. Thanks!

**Review by TomShawn — APPROVED** [review]
/label-require LGT1

File: dumpling-overview.md (hunk 3 above)
Comment (Joyinqin) [reply dumpling-overview.md]: Done in https://github.com/pingcap/docs/pull/5521

Comment (TomShawn) [reply dumpling-overview.md]: Thanks!

**Pull request merged**